### PR TITLE
support for LinuxMint on Debian base added

### PR DESCRIPTION
**Pull request by cehbrecht**
... mainly copied from existing code. Works on my LinuxMint machine.

Cheers,
Carsten

### CODE REVIEW

**Comment by ghost** [comment]
Test PASSed.
Refer to this link for build results: http://cookingwithsalt.org/job/bootstrap/job/pr/job/build/75/

File: bootstrap-salt.sh [hunk 1]
```diff
@@ -3576,7 +3581,7 @@ __configure_freebsd_pkg_details() {
     copyfile $conf_file /etc/pkg/FreeBSD.conf
     SALT_PKG_FLAGS="-r FreeBSD"
     ## ensure future ports builds use pkgng
-    echo "WITH_PKGNG=	yes" >> /etc/make.conf
+    echo "WITH_PKGNG=   yes" >> /etc/make.conf
```
Comment (s0undt3ch) [inline bootstrap-salt.sh]: Please revert this single line which is unrelated to the feature added. The remaining code looks awesome!

**Comment by ghost** [comment]
Test PASSed.
Refer to this link for build results: http://cookingwithsalt.org/job/bootstrap/job/pr/job/build/76/

**Comment by cehbrecht** [comment]
hopefully removed the whitespace change from the patch ...

**Comment by s0undt3ch** [comment]
Awesome!
